handler/instance_started.go

```diff
@@ -118,7 +118,12 @@ func (h *InstanceStarted) Handle(ctx context.Context, workerID int, msg kafka.Me
 		"num_variables": len(resp.Dataset.Variables.Edges),
 	})
 
-	ireq := h.createUpdateInstanceRequest(ctx, resp.Dataset.Variables, e, r.CantabularBlob, i.CodeLists)
+	if len(i.Editions) < 1 {
```

Comment:
```suggestion
	if len(i.Editions) == 0 {
```
Hopefully we cannot have negative number of editions and this will read better.

Ideally I would prefer to have a method 
```
func (i *Instance) IsEditionsEmpty() bool {
	if len(i.Editions) == 0 {
		return true
	}
	return false
}
```

either on the https://github.com/ONSdigital/dp-api-clients-go/ project or here...

Non blocker...